doc-botonic/docs/concepts/18.webchat.md

```diff
@@ -30,6 +30,156 @@ Botonic supports the following browsers:
 | Internet Explorer | 11 (with polyfills) |
 
 
+
+### How to make your bot compatible with IE11
+
+To make a bot compatible with IE11, its javascript code must be compliant with ES5. To do so, babel is used to transpile all the code needed to ES5.
+
+#### Prepare the bot
+
+To make the bot work in IE11:
+
+1. Add the following meta in the bot's integration code:
+
+```javascript 
+<meta http-equiv="X-UA-Compatible" content="IE=11" />
+```
+
+2. If style rendering issues occur, [disable webchat animations](/templates/template-custom-webchat):
+
+```javascript 
+theme: {
+  animations: {
+    enable: false,
+  },
+}
+```
+
+#### Add polyfills
+
+Some functionalities supported by most browsers are not implemented in IE11. To use them, polyfills are needed. 
+
+Here is a list of polyfills used in some of the bots:
+
+- Basic polyfills: [core-js](https://www.npmjs.com/package/core-js)
+- Style issues: [@webcomponents/webcomponentsjs](https://www.npmjs.com/package/@webcomponents/webcomponentsjs)
+- scrollTo feature: [element-scroll-polyfill](https://www.npmjs.com/package/element-scroll-polyfill)
+- normalize: [unorm](https://www.npmjs.com/package/unorm)
+
+To add polyfills:
+
+1. Open the `webpack.config.js` file.
+
+2. Add all the polyfills that are currently used in the entry property within the `botonicWebchatConfig`. The first 2 polyfills are mandatory, as well as the 2 that follows:
+
+```javascript
+entry: [
+  'core-js/stable/',
+  'regenerator-runtime/runtime',
+  '@webcomponents/webcomponentsjs',
+  'element-scroll-polyfill',
+  'unorm',
+  path.resolve('src', 'webchat-entry.ts'),
+],
+```
+
+#### Configure the Babel loader
+
+When configuring the Babel loader to transpile code to ES5 compliance, some options must be added/configured.
+
+1. Set `sourceType` to 'unambiguous' (cf. issue: https://github.com/babel/babel/issues/8900 )
+
+2. In presets, the definition order is important. Therefore, `babel/env` preset needs to be set before typescript to avoid losing constructor auto-assign: (https://github.com/babel/babel/issues/8752#issuecomment-486541662)
```

Comment:
@asastre but "constructor auto-assign" is not solved right?

Reply:
@dpinol True...this issue is also mentioned [here](https://github.com/hubtype/botonic/pull/909/files/6ca18eb8b0774a7a4bcde539a282ebc42eb54096#diff-6082cf5373720d9a9dac3599980abc0eR150). 
I wasn't able to find a proper solution for the "constructor auto-assign", neither this one nor the other one works.
I don't know if it will be better to remove this point here or add an explanation that even with this preset's order this issue is not solved.